test/rules/no-anonymous-default-export.spec.ts

```diff
@@ -1,12 +1,13 @@
+import { cjsRequire } from '@pkgr/core'
```

Comment:
I don't get it. `cjsRequire` uses a base dir from its own `import.meta.url`, not the current file's `import.meta.url`. How does this work?

Reply:
It's just for resolving packages, for relative paths, I inlined:

```ts
const cjsRequire = typeof require === 'undefined' ? createRequire(import.meta.url) : require
```